ios/NewExpensify.xcodeproj/project.pbxproj

```diff
@@ -750,7 +750,7 @@
 				COPY_PHASE_STRIP = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				ENABLE_TESTABILITY = YES;
-				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = i386;
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "";
```

Comment:
Can the entire line be removed instead?

Reply:
Maybe, but it's helpful to keep things as in-sync with the React Native example app as possible for the sake of using the Upgrade Helper: https://react-native-community.github.io/upgrade-helper/?from=0.70.1&to=0.70.4